agents/official/composition/src/composition/configuration.py

```diff
@@ -0,0 +1,12 @@
+from pydantic import AnyUrl, computed_field
+from pydantic_settings import BaseSettings
+
+
+class Configuration(BaseSettings):
+    platform_url: AnyUrl = "http://localhost:8333"
```

Comment:
I'd unify this everywhere under `BEEAI_HOST` (including `beeai-server`, `beeai-cli` and other projects)